src/azure-cli/azure/cli/command_modules/resource/custom.py

```diff
@@ -403,7 +403,7 @@ class JsonCTemplatePolicy(SansIOHTTPPolicy):
 
     def on_request(self, request):
         http_request = request.http_request
-        logger.info(http_request.data)
+        logger.debug(http_request.data)
```

Comment:
Won't debug also "disclosure of the value for confidential parameters"?

Reply:
It is the same as the request body printed in log `cli.azure.cli.core.sdk.policies`, so it's redundant, I deleted it